owa/app/css/openmrs-owa-idgen.css

```diff
@@ -117,21 +118,205 @@ nav{
     background-color: #009384 !important;
     border-color: #ddd #ddd #fff;
 }
 
+.clear {
+    clear: both
+}
+
+
+/* Input areas */
+
+
+/* Table styles*/
+
+.logwrapper .ReactTable {
+    margin: 20px;
+}
+
+.ReactTable .rt-resizable-header.-cursor-pointer.rt-th {
+    border-bottom: 1px solid #dddddd;
+    background: #f3f3f3;
+    width: auto;
+    text-align: left;
+    word-break: normal;
+    font-weight: bold;
+    padding: 10px 9px;
+}
+
+.rt-tr.-even {
+    background: #fffdf7;
+}
+
+.logwrapper {
+    font-family: OpenSans
+}
+
+.logwrapper .form-control,
+select.form-control {
+    border-radius: 0px;
+    line-height: 1.7em;
+}
+
+.logSearch {
+    background: #3f3d74;
+    text-align: center;
+    color: white;
+    padding: 20px 30px;
+    -moz-border-radius: 5px;
+    -webkit-border-radius: 5px;
+    -o-border-radius: 5px;
+    -ms-border-radius: 5px;
+    -khtml-border-radius: 5px;
+    border-radius: 5px;
+    max-width: 450px;
+    margin: 0px auto 10px
+}
+
+.search_label {
+    color: #363463;
+    text-align: center;
+    line-height: 2em;
+    font-size: 1.2em;
+}
 
+.search_label span {
+    font-family: "OpenSansLight";
+    padding-left: 10px
+}
+
+input.log_searchbox {
+    margin-top: 0px;
+    font-size: 100%;
+    font-family: inherit;
+    line-height: 1.7em;
+}
+
+.searchlink {
+    background: #3f3d74;
+    color: #fffdf7;
+    border: 1px solid #fffdf7;
+    padding: 10px;
+    margin: 10px auto 0px;
+}
+
+.searchlink:hover {
+    color: #e8e4d8;
+    background: #3f3d74;
+}
+
+.advanced_wrapper fieldset>div {
+    float: left;
+    margin-bottom: 10px;
+}
+
+.advanced_wrapper fieldset input,
+select {
+    width: 100%
+}
+
+.logs_table_area {
+    position: relative;
+}
+
+.advanced_wrapper,
+.logs_table {
+    width: 100%;
+    left: 0;
+}
+
+.advanced_wrapper {
+    padding: 20px 20px 30px;
+    z-index: 10;
+    position: absolute;
+    background: rgba(255, 255, 255, 0.95);
+    border: 1px solid #3f3d74;
+    border-top-width: 5px;
+    margin-top: -15px;
+    text-align: center;
+}
+
+label.search_lbl {
+    margin-bottom: 0px
+}
+
+button {
+    background-color: #dddddd;
+    border: #dddddd 1px solid;
+    padding: 8px 20px;
+    display: inline-block;
+    line-height: 1.2em;
+    color: #363463;
+    cursor: pointer;
+    min-width: 0;
+    max-width: 300px;
+    text-decoration: none;
+    -moz-border-radius: 3px;
+    -webkit-border-radius: 3px;
+    -o-border-radius: 3px;
+    -ms-border-radius: 3px;
+    -khtml-border-radius: 3px;
+    border-radius: 3px;
+    background: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #ffffff), color-stop(100%, #dddddd));
+    background: -webkit-linear-gradient(top, #ffffff, #dddddd);
+    background: -moz-linear-gradient(top, #ffffff, #dddddd);
+    background: -o-linear-gradient(top, #ffffff, #dddddd);
+    background: -ms-linear-gradient(top, #ffffff, #dddddd);
+    background: linear-gradient(top, #ffffff, #dddddd);
+}
+
+button:hover {
```

Comment:
select your button by it's id

Reply:
Sorted, but think I should leave the styles to handle browser rendered styles for the:
-  input areas (_textarea, input, button,select_): to remove the outline on focus
- a: to remove the blueish color when the link is clicked or has been visited in Chrome